src/Pomodoro.Web/wwwroot/css/app.css

```
@@ -178,6 +225,43 @@ input, textarea, [contenteditable="true"] {
+    right: 0;
+    background: var(--color-background-primary);
+    border-top: 0.5px solid var(--color-border-tertiary);
+    z-index: 100;
+    padding: 6px 0;
+    padding-bottom: calc(6px + env(safe-area-inset-bottom, 0px));
+}
+
+    .app-content {
+        padding-bottom: 60px;
+    }
+}
```

Stylelint (17.9.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)